code/game/objects/structures/fluff.dm

```diff
@@ -49,3 +49,9 @@
 	desc = "Although comfortable, this sleeper won't function as anything but a bed ever again."
 	icon = 'icons/obj/lavaland/spawners.dmi'
 	icon_state = "cryostasis_sleeper_open"
+
+/obj/structure/fluff/broken_flooring
+	name = "broken tiling"
+	desc = "A segment of broken flooring"
+	icon = 'icons/obj/brokentiling.dmi'
```

Comment:
The floor tile sprite is exactly the same as the one we used because that's what I used as a base. The darker outlining on the tiles themselves are to help differentiate between the singular tiles and give some extra depth.

I'll see to aligning the plating properly but this is meant to scattered about on the edge of broken/ripped apart station segments, not really mass-used. It adds a nice extra bit of detail for debris that otherwise looks lack-luster with the regular items.